File: petpals-pals/src/main/java/com/petpals/bootstrap/quarkusapp/Pals.java
package com.petpals.bootstrap.quarkusapp;

import io.quarkus.runtime.Quarkus;
import io.quarkus.runtime.annotations.QuarkusMain;



@QuarkusMain
public class Pals {
    public static void main(String ... args) {
        Quarkus.run(args);
    }
}

File: petpals-pals/src/main/kotlin/com/petpals/bootstrap/Pals.kt
package com.petpals.bootstrap

import io.quarkus.runtime.Quarkus
import io.quarkus.runtime.annotations.QuarkusMain

@QuarkusMain
object Pals {
    @JvmStatic
    fun main(args: Array<String>) {
        Quarkus.run(*args)
    }
}